README.md

```diff
@@ -0,0 +1,229 @@
+# Nushell RFCs
+
+[Nushell RFCs]: #nushell-rfcs
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Though some changes are substantial enough that we ask for these to be
+put through a bit of a design process and produce a consensus among the
+Nushell community.
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for new features to be added to Nushell, so that all
+stakeholders can be confident about the direction the project is evolving in.
+
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#nushell-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
```

Comment:
need → should, until the note below is removed? Or will we change to "need to" before merging?

Reply:
Good catch. I had changed the header, but missed this ToC entry. Fixed!